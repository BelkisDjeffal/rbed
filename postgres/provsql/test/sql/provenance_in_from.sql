\set ECHO none
SET search_path TO provsql_test,provsql;

select * from provenance(), personnel;
