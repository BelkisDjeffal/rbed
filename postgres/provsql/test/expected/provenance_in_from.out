\set ECHO none
ERROR:  provenance() called on a table without provenance
